app/models/course/experience_points_disbursement.rb

```diff
@@ -10,23 +10,23 @@ class Course::ExperiencePointsDisbursement
   attr_accessor :reason
 
   # @!attribute [rw] course
-  #   The course that this disbursement is for.
+  #   The course that this disbursement is for. This attribute is read during authorization.
   #   @return [Course]
   attr_accessor :course
 
-  # @!attribute [r] experience_points_records
-  #   Experience points records that will potentially be created.
-  #   @return [Array<Course::ExperiencePointsRecords>]
-  attr_reader :experience_points_records
+  # @!attribute [rw] group_id
+  #   ID of the group that this disbursement is for. nil is returned if no group is specified.
+  #   @return [Integer]
```

Comment:
Just a check, it is also possible that nil is returned if there is no group specified?